client/services/SolutionHandlerServiceSpec.js

```diff
@@ -353,10 +353,8 @@ describe('SolutionHandlerService', function() {
           auxiliaryCode, 'python'
         ).then(function(feedback) {
           expect(feedback.getParagraphs()[0].getContent()
-            .startsWith(
-              'Looks like your code had a runtime error when ' +
-              'evaluating the input ')
-          ).toBe(true);
+            .startsWith('It looks like greeting isn\'t a declared variable.')
+            ).toBe(true);
```

Comment:
deindent by 2 so that ')' matches level of '('. 

(Might need to break after initial '('.)